src/main/java/io/kestra/plugin/aws/glue/StopJobRun.java

```diff
@@ -0,0 +1,181 @@
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Stop a running AWS Glue job and optionally wait for completion."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Stop a Glue job and wait for its completion",
+            full = true,
+            code = """
+                id: stop_glue_job
+                namespace: company.team
+
+                tasks:
+                  - id: stop
+                    type: io.kestra.plugin.aws.glue.StopJobRun
+                    accessKeyId: "{{ secret('AWS_ACCESS_KEY_ID') }}"
+                    secretKeyId: "{{ secret('AWS_SECRET_KEY_ID') }}"
+                    jobName: my-glue-job
+                    jobRunId: jr_1234567890abcdef
+                    wait: true
+                    interval: PT1S
+                """
+        )
+    }
+)
+public class StopJobRun extends AbstractGlueTask implements RunnableTask<Output> {
+
+    @Schema(title = "The name of the Glue job to stop")
+    @PluginProperty(dynamic = true)
+    @NotNull
+    private Property<String> jobName;
+
+    @Schema(title = "The ID of the job run to stop")
+    @PluginProperty(dynamic = true)
+    @NotNull
+    private Property<String> jobRunId;
+
+    @Schema(
+        title = "Wait for the job to be fully stopped before ending the task.",
+        description = "If true, the task will periodically check the job status until it reaches a stopped state."
+    )
+    @Builder.Default
+    private Property<Boolean> wait = Property.ofValue(true);
+
+    @Schema(
+        title = "Interval between status checks"
+    )
+    @Builder.Default
+    private Property<Duration> interval = Property.ofValue(Duration.ofSeconds(1));
+
+    @Override
+    public Output run(RunContext runContext) throws Exception {
+        // Render properties once
+        String jobNameValue = runContext.render(this.jobName).as(String.class).orElseThrow();
+        String jobRunIdValue = runContext.render(this.jobRunId).as(String.class).orElseThrow();
+        boolean waitValue = runContext.render(this.wait).as(Boolean.class).orElse(true);
+        Duration intervalValue = runContext.render(this.interval).as(Duration.class).orElse(Duration.ofSeconds(1));
+
+        try (GlueClient glueClient = this.client(runContext)) {
+            // Stop the job
+            BatchStopJobRunResponse stopResponse = glueClient.batchStopJobRun(
+                BatchStopJobRunRequest.builder()
+                    .jobName(jobNameValue)
+                    .jobRunIds(jobRunIdValue)
+                    .build()
+            );
+
+            runContext.logger().info("Stopped Glue job '{}' with run ID: {}", jobNameValue, jobRunIdValue);
+
+            GetJobRunRequest getJobRunRequest = createGetJobRunRequest(jobNameValue, jobRunIdValue);
+            AtomicReference<software.amazon.awssdk.services.glue.model.JobRun> currentJobRun = new AtomicReference<>();
```

Comment:
Could you remove all the `import software.amazon.awssdk.services*` package from the code and import properly the classes please?

```suggestion
            AtomicReference<JobRun> currentJobRun = new AtomicReference<>();
```